src/ES6/Modules.js

```diff
@@ -0,0 +1,16 @@
+// const getName = () => {
+//   return 'Simone';
+// };
+
+// exports.getName = getName;
+
+//↑ This example will work because it use common JS
```

Comment:
Remove unnecessary comments to keep this file neat

Reply:
I have removed it